.gitignore

```diff
@@ -1,2 +1,10 @@
 .DS_Store
-.idea
\ No newline at end of file
+.idea
+out.txt
+1.txt
```

Comment:
do not ignore each file individually, just store them under a special directory and ignore the whole directory instead.